docs/filter_descriptions.md

```diff
@@ -12,19 +12,21 @@
 8.  [Crop Width (CropW)](#cropw)
 9.  [Crop Height (CropH)](#croph)
 10.  [Peak Signal to Noise Ratio (PSNRf)](#psnrf)
-11.  [Structural SImilarity Metric (SSIMf)](#ssimf)
-12.  [Mean Square Error (MSEf)](#msef)
-13.  [Interlacement Detection (single frame)](#singleframe)
-14.  [Interlacement Detection (multiple frames)](#multiframe)
-15.  [Interlacement Detection (repeated fields)](#repeated)
+11.  [Packet Duration Time](pkt_duration_time)(#pkd)
```

Comment:
The parenthesis here and below are causing a disruption in the linking, so the links break and (#pkd) shows up. The parenthesis should live inside of the brackets.